app/src/main/java/at/roteskreuz/stopcorona/model/repositories/BluetoothRepository.kt

```diff
@@ -0,0 +1,28 @@
+package at.roteskreuz.stopcorona.model.repositories
+
+import android.app.PendingIntent
+import android.bluetooth.BluetoothAdapter
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.ActivityCompat.startActivityForResult
+
+/**
+ * wrapper for Bluetooth related functionality
+ */
+class BluetoothRepository(
+    private val bluetoothAdapter: BluetoothAdapter
+
+){
+    fun isBluetoothEnabled(): Boolean {
+        return bluetoothAdapter.isEnabled
+    }
+
+    fun enableBluetoothIntent(): Intent {
```

Comment:
I would maybe go with a way of clear getter
```
val enableBluetoothIntent: Intent = Intent(BluetoothAdapter.ACTION_REQUEST_ENABLE)
```
or using `get` as a prefix to the method name.

Same for `ExposureNotificationRepository` new methods.